From: SeongJae Park <sj@kernel.org>
Date: Mon, 14 Oct 2024 16:50:06 -0700
Subject: [PATCH] Revert "selftests/damon/config: remove configs for DAMON
 debugfs interface selftests"

This reverts commit a24e72ffe812a9509632053b3af62cb795aa428c.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 tools/testing/selftests/damon/config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/testing/selftests/damon/config b/tools/testing/selftests/damon/config
index a68a9fead5dc..0daf38974eb0 100644
--- a/tools/testing/selftests/damon/config
+++ b/tools/testing/selftests/damon/config
@@ -1,5 +1,6 @@
 CONFIG_DAMON=y
 CONFIG_DAMON_SYSFS=y
+CONFIG_DAMON_DBGFS=y
 CONFIG_DAMON_PADDR=y
 CONFIG_DAMON_VADDR=y
 CONFIG_DAMON_RECLAIM=y
-- 
2.39.5

